docs/docsource/installation/install/Install_Zenko.rst

```diff
@@ -24,7 +24,10 @@ Follow these steps to install Zenko with ingress.
 
 .. _create_options.yaml:
 
-1. Create an options.yaml file in Zenko/kubernetes/ to store deployment
+Final Preparations
```

Comment:
Maybe name this `Configuration Setup`/`Setup Configuration`? Having the first section named Final Preparations could throw people off

Reply:
Changed to "Configure with options.yaml" 

(The only reason I added these headings was to give the cross-references something to push against...)